docs/reference/scripting.md

```diff
@@ -340,7 +340,9 @@ packages. For more details on this, see the
 
 ## Operators
 The following table lists all available unary and binary operators with effect,
-arity (unary, binary) and precedence level (higher binds stronger).
+arity (unary, binary) and precedence level (higher binds stronger). Some
+operations, such as [modulus]($calc.rem), do not have a special syntax and can 
```

Comment:
I think modulus and remainder [are technically different operations](https://stackoverflow.com/questions/13683563/whats-the-difference-between-mod-and-remainder).

Reply:
Would linking to [`rem-euclid`](https://typst.app/docs/reference/foundations/calc#functions-rem-euclid) be more appropriate?

Reply:
Yes, I think so